CMakeLists.txt

```diff
@@ -471,6 +471,9 @@ OCV_OPTION(WITH_OBSENSOR "Include obsensor support (Orbbec RGB-D modules: Astra+
 OCV_OPTION(WITH_CANN "Include CANN support" OFF
   VISIBLE_IF TRUE
   VERIFY HAVE_CANN)
+OCV_OPTION(WITH_FLATBUFFERS "Include FlatBuffers support" OFF
```

Comment:
Have a concern here: people would not know they need to turn it on for the support of TFLite models, because there are no connections in terms of their names.

Also is it possible to bring flatbuffers into 3rdparty like protobuf so that we can distribute pre-built packages with this support?

Reply:
I added a corresponding exception message when user tries load TFLite model. For Caffe, TF and ONNX the same flag is WITH_PROTOBUF (however enabled by default).

> Also is it possible to bring flatbuffers into 3rdparty like protobuf so that we can distribute pre-built packages with this support?

It's possible but I don't know if it's suitable to add a new 3rdparty dependency just for TFLite.

Reply:
@dkurt I believe we could start with the current state and add 3rdparty later as a separate activity.
Could you please add information into PR's description about required build environment configuration:
- Ubuntu version, used packages names and versions?
- what is needed to build OpenCV.js through Emscripten?

Reply:
> I added a corresponding exception message when user tries load TFLite model.

Okay, that makes sense. Adding as 3rdparty can be done later.

Reply:
@alalek, added, thanks! Emscripten command is turned pretty smooth.